documentation/docs/13-configuration.md

```diff
@@ -135,6 +136,10 @@ When pages are prerendered, the CSP header is added via a `<meta http-equiv>` ta
 
 > When `mode` is `'auto'`, SvelteKit will use nonces for dynamically rendered pages and hashes for prerendered pages. Using nonces with prerendered pages is insecure and therefore forbidden.
 
+### endpointExtensions
+
+An array of valid file extensions for endpoint modules.
```

Comment:
"Valid" seems like a strange word to be using here to me. It's not like Kit will throw an error for other ones, will it? Just ignore them. I don't have another word to suggest at the moment.

Reply:
```suggestion
An array of file extensions that SvelteKit will treat as endpoints. Files with extensions that match neither `config.extensions` nor `config.kit.endpointExtensions` will be ignored by the router.
```